ui/src/app/utils/strings.ts

```diff
@@ -29,3 +29,12 @@ export const oxfordCommaString = (
       return `${allButLast}, and ${last}`;
     }
   );
+
+export const toPascalCase = (str: string) => {
```

Comment:
thinking out loud here (does not block merge) - this might be nicer to include inside `fromUserAppStatusWithFallback()` ?  Actually it would be good to have a standardized way of displaying a variety of things, like AppType, AppStatus, RuntimeStatus, etc. 

anyway, future idea

Reply:
I like this idea, i can put a new PR for the same